articles.tex

```diff
@@ -258,6 +268,18 @@
 The resignation will be read at the following House Meeting and become effective at that time.
 \asubsection{Advisory Membership Term}
 Advisory Membership shall last until the member resigns.
+\asection{Leave of Absence}
+A leave of absence offers the option for members to take extended time away from their responsibilities to House for any number of personal issues including, but not limited to, physical illness, mental illness, or care giving for a sick family member. Computer Science House recognizes the need for its members to take care of their personal issues and will support them through the process of requesting a leave of absence as well as acclimating back to the culture of House upon their return. Upon approval of a leave of absence request, the Evaluations Director is notified and the member's leave begins immediately. A member is allowed to be physically present and also be on a leave of absence. The leave of absence can also be applied retroactively to the member at the discretion of the Evaluations Director.
+\asubsection{Leave of Absence Request}
+A leave of absence request must include the following information about the member: name, email, phone number, start date, expected return date, and a reason for the leave. The request is then given to a staff member of Residence Life, excluding student employees, for approval.
```

Comment:
The request can be given to any staff member of ResLife, excluding student employees. So this includes people like Aaron and Adam as well as any other adult in ResLife the member may feel comfortable with, excluding RAs. 

Additionally, the criteria for approval is established by ResLIfe.